app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,24 @@
  *= require_tree .
  *= require_self
  */
+
+.flash {
+  -moz-border-radius: 4px 4px 4px 4px;
+  -webkit-border-radius: 4px 4px 4px 4px;
+  border-radius: 4px 4px 4px 4px;
+  margin-bottom: 18px;
+  margin-top: 18px;
+  padding: 12px 25px 12px 24px;
+  -moz-box-shadow: 3px 3px 1px 1px #CCCCCC;
+  -webkit-box-shadow: 3px 3px 1px 1px #CCCCCC;
+  box-shadow: 3px 3px 1px 1px #CCCCCC;
+  color: white;
+}
```

Comment:
Just because you said you were interested:

The browser prefixes are a good instinct, but it's worth checking whether they're still useful for the directives you're using. For example, `box-shadow` — check this page out and select the 'Usage Relative' button on the left hand side: http://caniuse.com/#search=box-shadow

You'll see that guarding against old firefox (`-moz-`) and chrome/safari (`-webkit-`) browsers doesn't really buy you anything. Essentially everyone using those browsers is using versions that don't need browser prefixes.

Also, when you do have to use them, they're usually formatted like this:

```css
-webkit-box-shadow: 3px 3px 1px 1px #CCCCCC;
   -moz-box-shadow: 3px 3px 1px 1px #CCCCCC;
        box-shadow: 3px 3px 1px 1px #CCCCCC;
```

Also worth noting there are libraries that can do all this work for you, rather than you having to remember it all, just in case you end up working with them again!